ui/src/components/core/combobox/dropdown.tsx

```diff
@@ -0,0 +1,81 @@
+import { Fragment } from 'react';
+import { Listbox, Transition } from '@headlessui/react';
+import { Icon } from '@/components/core/icon';
+import { Flex } from '@/components';
+
+export type DropdownItem = {
+  value: string;
+  label: string;
+};
+
+export type DropdownProps = {
+  items: DropdownItem[];
+  selectedValue: DropdownItem | undefined;
+  onChange(option: DropdownItem): void;
+};
+
+export const Dropdown: React.FC<DropdownProps> = ({

```

Comment:
The same comments from previous component, we could split this in smaller components and move the handlers.